Sources/Foundation/Process.swift

```diff
@@ -932,13 +932,6 @@ open class Process: NSObject, @unchecked Sendable {
         var spawnAttrs: posix_spawnattr_t? = nil
 #else
         var spawnAttrs: posix_spawnattr_t = posix_spawnattr_t()
-#endif
-#if os(Android)
-        guard var spawnAttrs else {
```

Comment:
I don't know what the point of this is, as it's set to nil just a couple lines above and fails to compile with NDK 26.